src/common/models/project.go

```diff
@@ -120,11 +124,71 @@ func (p *Project) AutoScan() bool {
 	return isTrue(auto)
 }
 
+// QueryPublic ...
+func (p *Project) QueryPublic(qs orm.QuerySeter, key string, value interface{}) orm.QuerySeter {
```

Comment:
PublicQuerySeter()?

Reply:
`FilterPublic` , `FilterMember`, `FilterOwner` or `FilterByPublic`, `FilterByMember`, `FilterByOwner` ?

`XxxQuerySeter` not very match the intention of the function.

Reply:
Changed to use `FilterByXxx`